index.bs

```diff
@@ -969,6 +969,7 @@ To <dfn>populate the pose</dfn> of an {{XRSpace}} |space| in an {{XRSpace}} |bas
   1. If |space|'s [=XRSpace/session=] does not equal |session|, throw an {{InvalidStateError}} and abort these steps.
   1. If |baseSpace|'s [=XRSpace/session=] does not equal |session|, throw an {{InvalidStateError}} and abort these steps.
   1. Check if [=poses may be reported=] and, if not, throw a {{SecurityError}} and abort these steps.
+  1. If [=poses must be limited=] between |space| and |baseSpace|, return <code>null</code>.
```

Comment:
I suspect we should follow the tracking loss behavior here rather than just returning null? @thetuvix

Reply:
Updated this to make hitting the limit act more like tracking loss.